### PR TITLE
Monkey transformations now properly carry over your implants.

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Fixes the code for carrying over implants when transforming into a monkey/terratoma or back into a human

## Why It's Good For The Game
Code actually working is good, plus it enables more traitor tot shenanigans.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://user-images.githubusercontent.com/110184118/224981571-61f6c6c5-2b1b-4dd1-9c7e-cf969309dfc8.mp4

</details>

## Changelog
:cl:
fix: implants are now properly carried over during the monkey-human transformation
/:cl:

### CODE REVIEW

**Comment by CydiaLamiales** [comment]
Linters, I swear to god...

**Comment by Fronsis** [comment]
A little relevant to the PR but, How hard would be to apply this to transformation sting as well? Althought very rare one time an antag prepared a show and had a bomb in them but due to transformation sting a ling did 4thelols they lost the bomb and lost the explosive finale due to being another race

File: code/modules/mob/transform_procs.dm [hunk 1]
```diff
@@ -74,11 +66,10 @@
...
 		O.radiation = radiation
 
-	//re-add implants to new mob
-	if (tr_flags & TR_KEEPIMPLANTS)
-		for(var/Y in implants)
-			var/obj/item/implant/IMP = Y
-			IMP.implant(O, null, 1)
+	//move implants to new mob
+	if(tr_flags & TR_KEEPIMPLANTS)
+		for(var/obj/item/implant/IMP in implants)
```
Comment (Penwin0) [inline code/modules/mob/transform_procs.dm]: You can use `as()` or `as anything` in these for loops, as `implants` should never have anything that isn't `/obj/item/implant/`

Comment (CydiaLamiales) [reply code/modules/mob/transform_procs.dm]: Yes but transfer_implant is a proc on obj/item/implant

Comment (CydiaLamiales) [reply code/modules/mob/transform_procs.dm]: Updated it

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**